test/extensions/filters/network/thrift_proxy/integration.cc

```diff
@@ -80,6 +80,10 @@ void BaseThriftIntegrationTest::preparePayloads(const PayloadOptions& options,
     args.push_back(absl::StrJoin(headers, ","));
   }
 
+  auto temp_path = TestEnvironment::temporaryDirectory();
+  args.push_back("-T");
+  args.push_back(temp_path);
```

Comment:
Is there much value in adding the '-T' param instead of keeping this detail internal to generate_fixture.sh? Or maybe it was just the easiest way to get a tmp dir on all platforms?

Reply:
`TestEnvironment::temporaryDirectory()` is a bit involved in the way it generates the temporary directory see here https://github.com/envoyproxy/envoy/blob/0b24c6ac8ebaddfd46852e732d82675b3521d35a/test/test_common/environment.cc#L70

As a result there might be a case that the temporary directory that the `generate_fixture.sh` just uses a different temp directory than the test code. This leads to crashes when the test code is trying to read the `request/response` files

Reply:
Got it